main.go

```diff
@@ -46,6 +46,18 @@ func main() {
 		log.Fatalf("error reading config: %v", err)
 	}
 
+	for k, ac := range cfg.Agents {
+		if k != "java" && k != "nodejs" {
+			if ac.Image == "" {
+				log.Fatalf("custom agent %q is missing Image", k)
+			}
+
+			if ac.ArtifactPath == "" {
+				log.Fatalf("custom agent %q is missing Artifact", k)
+			}
+		}
+	}
```

Comment:
```suggestion
	for k, ac := range cfg.Agents {
		if ac.Image == "" {
			log.Fatalf("agent %q is missing 'image'", k)
		}
		if ac.ArtifactPath == "" {
			log.Fatalf("agent %q is missing 'artifact'", k)
		}
	}
```

Do we need to special case the config names? They are defined in [Helm template values](https://github.com/elastic/apm-mutating-webhook/blob/869bc6349f84b132e8904712ed0facb2522d8aa1/apm-agent-auto-attach/values.yaml#L26-L37), and in theory could be overridden IIANM. I think we should check if the image/artifact name are unspecified regardless of the config name.

Also I suggest using the lowercase names as they should appear in custom.yaml.

Reply:
Finally, maybe move this validation to `parseConfig` so it can be unit tested?

Reply:
Good point!